src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -26,16 +26,16 @@ if (PHP_VERSION_ID >= 70200) {
     $PHPUNIT_VERSION = '4.8';
 }
 
-if (file_exists(($path = dirname(__DIR__)).'/composer.json')) {
-    if (file_exists($path.'/vendor/symfony/phpunit-bridge/composer.json')) {
-        $path .= '/vendor/symfony/phpunit-bridge';
+$root = __DIR__;
+while (!file_exists($root.'/composer.json') || file_exists($root.'/bin/simple-phpunit')) {
```

Comment:
@fbourigault do you know what the second part of the if is about?

Reply:
We are going up in directories until we find a composer.json or a bin/simple-phpunit. IMHO this is a special case for Symfony monolithic repository.

Reply:
Thanks for answering, I got it straight from the horse's mouth in the meantime but forgot I asked you.